input/core/src/commonMain/kotlin/majestic/editor/body/chunks/EditorControl.kt

```diff
@@ -0,0 +1,8 @@
+package majestic.editor.body.chunks
+
+import majestic.editor.insert.InsertHostController
+
+data class EditorControl(
+    val actions: InsertHostController,
+    val dropDowns: InsertHostController
+) : Chunk
```

Comment:
An editor control is not a chunk (as per our definition). Remove it from the Chunk Hierarchy